packages/notifi-react-card/lib/utils/NotificationHistoryUtils.tsx

```diff
@@ -242,8 +227,29 @@ const getAlertDetailsContents = (
       };
 };
 
+const concatHistoryNodes = (
+  nodes: NotificationHistoryEntry[],
+  nodesToConcat: NotificationHistoryEntry[],
+) => {
+  switch (nodes[0]?.__typename) {
+    case 'FusionNotificationHistoryEntry':
+      if (nodesToConcat[0]?.__typename !== 'FusionNotificationHistoryEntry') {
+        throw new Error('Type mismatch: FusionNotificationHistoryEntry');
```

Comment:
rather than throwing, can we just return `nodes` here? (i.e. throw away nodesToConcat)

Reply:
Nice idea👍. modified.